docs/guides/platform/DevWorkspace_operator_CRDs.md

```diff
@@ -0,0 +1,124 @@
+# DevWorkspace operator CRDs
+
+## DevWorkspaceOperatorConfig/v1alpha1
+
+DevWorkspaceOperatorConfig is the schema for the <span translate="no">devworkspaceoperatorconfigs</span> API.
+
+### config
+
+**Description:** OperatorConfiguration defines configuration options for the DevWorkspace Operator.
+
+**Type:**  object
+
+| Property                          | Type        | Description                       |
+|-----------------------------------|-------------|-----------------------------------|
+|config                             |object       |Defines configuration options for the DevWorkspace Operator.|
+|config.enableExperimentalFeatures  |boolean      |EnableExperimentalFeatures turns on in-development features of the controller. This option should generally not be enabled, as any capabilities are subject to removal without notice.|
+|config.routing                     |object       |Routing defines configuration options related to DevWorkspace networking|
+|config.routing.clusterHostSuffix   |string       |ClusterHostSuffix is the hostname suffix to be used for DevWorkspace endpoints. On OpenShift, the DevWorkspace Operator will attempt to determine the appropriate value automatically. Must be specified on Kubernetes.|
+|config.routing.defaultRoutingClass |string       |DefaultRoutingClass specifies the routingClass to be used when a DevWorkspace specifies an empty `.spec.routingClass`. Supported routingClasses can be defined in other controllers. If not specified, the default value of "basic" is used.|
+|config.routing.proxyConfig         |object       |ProxyConfig defines the proxy settings that should be used for all DevWorkspaces. These values are propagated to workspace containers as environment variables.. On OpenShift, the operator automatically reads values from the \"cluster\" proxies.config.openshift.io object and this value only needs to be set to override those defaults. Values for httpProxy and httpsProxy override the cluster configuration directly. Entries for noProxy are merged with the noProxy values in the cluster configuration. To ignore automatically read values from the cluster, set values in fields to the empty string (\"\"). Changes to the proxy configuration are detected by the DevWorkspace Operator and propagated to DevWorkspaces. However, changing the proxy configuration for the DevWorkspace Operator itself requires restarting the controller deployment.|
+|config.routing.proxyConfig.httpProxy|string      |HttpProxy is the URL of the proxy for HTTP requests, in the format http://USERNAME:PASSWORD@SERVER:PORT/. To ignore automatically detected proxy settings for the cluster, set this field to an empty string ("").|
+|config.routing.proxyConfig.httpsProxy|string     |HttpsProxy is the URL of the proxy for HTTPS requests, in the format http://USERNAME:PASSWORD@SERVER:PORT/. To ignore automatically detected proxy settings for the cluster, set this field to an empty string ("").|
```

Comment:
Not very familiar with this configuration stuff, but is `http://USERNAME:PASSWORD@SERVER:PORT/` supposed to use `https://`... or is it intended to be the same format as the previous row?

Reply:
It should be https.